<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCHWInfo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Click File and then Get Info to get hardware details</string>
    </property>
   </widget>
   <widget class="QWidget" name="infoWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QListWidget" name="componentList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>291</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CPU</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memory</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Disk</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Network</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bios</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MotherBoard</string>
      </property>
     </item>
    </widget>
    <widget class="QTextBrowser" name="comonentBrowser">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGet_Info">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGet_Info"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuGet_Info"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGet_Info">
   <property name="text">
    <string>Get Info</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
